src/NuGet.Clients/NuGet.PackageManagement.UI/Resources/Resources.xaml.cs

```diff
@@ -13,7 +17,10 @@ public SharedResources()
         private void PackageIconImage_ImageFailed(object sender, ExceptionRoutedEventArgs e)
         {
             var image = sender as Image;
-            image.Source = Images.DefaultPackageIcon;
+
+            e.Handled = true; // don't repropagate the event
+            var binding = BindingOperations.GetBindingExpression(image, Image.SourceProperty);
```

Comment:
Use the explicit type instead of `var`.

Also [`BindingOperations.GetBindingExpression(…)`](https://docs.microsoft.com/en-us/dotnet/api/system.windows.data.bindingoperations.getbindingexpression?view=netframework-4.8#returns) says `null` could be returned.  Is this a possibility we should guard against?